### ISSUE TITLE
Static maps

### ISSUE THREAD

**Issue by ifyouwillit** [WordImpress/google-maps-builder#110]
Would be awesome to have an option for static maps too. Occasionally easier to load and better for usability. Especially on mobile.

**Comment by DevinWalker** [comment]
Good call. Seems like a good Pro feature. https://developers.google.com/maps/documentation/staticmaps/intro

**Comment by ifyouwillit** [comment]
Agree. I use statics on my websites and generate them with the google api programmatically. Would be much easier to implement at scale with this plugin.
